src/functions/hskip.js

```diff
@@ -0,0 +1,65 @@
+//@flow
+// Horizontal spacing commands
```

Comment:
Maybe call this file `kern.js` b/c `type: "kern"` below... or maybe `horizontal-spacing.js`.

Reply:
Good point -- switched to `kern.js`.